contents/tracks.mdx

```diff
@@ -4,100 +4,225 @@ showTitle: false
 showSidebar: true
 width: md
 images:
-    - './images/startup-hogs.png'
+    - './images/tracks/ask-iwata.jpg'
+    - './images/tracks/sand-hill-road.jpg'
+    - './images/tracks/messy-middle.jpg'
+    - './images/tracks/high-output.jpg'
+    - './images/tracks/no-rules-rules.jpg'
+    - './images/tracks/getting-real.png'
+    - './images/tracks/continuous-discovery.jpg'
+    - './images/tracks/mom-test.jpg'
+    - './images/tracks/founding-sales.jpg'
+    - './images/tracks/saas-playbook.jpg'
+    - './images/tracks/everyday-design.jpg'
+    - './images/tracks/from-switzerland.jpg'
 ---
 
 import { StaticImage } from 'gatsby-plugin-image'
 import { getImage, GatsbyImage } from 'gatsby-plugin-image'
 import CommunityCTA from 'components/CommunityCTA'
+import { FeatureSnapshot } from 'components/FeatureSnapshot'
+import writer from './images/blog/search-hog-4.png'
 
 <Hero
     title="PostHog Tracks"
-    subtitle="Curated lesson tracks from our engineers and users, to help you build better products"
+    subtitle="Building a startup isn't easy. But we can put you on the right track!"
 />
+<h3 className="text-xl m-0 font-semibold opacity-75 text-center !pb-8">
+Tracks is a curated reading list for founders building product-led companies like PostHog. We build in the open, so we're happy to share all the details of how we've got this far, and the lessons we learned on the way.</h3>
 
-PostHog Tracks is a curated course of tutorials and other lessons which cover common uses for particular roles, as well as general advice for all users - and it's constantly expanding!
+<h3 className="mb-0">How to <span style='color: #f64e00;'>hire a team</span> of world-class talent</h3>
 
-<p class="mt-4 pb-12 flex space-x-2 items-center justify-center">
-<CallToAction href="/tutorials">Browse all tutorials</CallToAction> 
-<CallToAction href="/questions" type="outline">Ask a question</CallToAction>
-</p>
-
-<h3 className="mb-0">PostHog 101</h3>
-
-Want a quick summary of the essential info? Look no further than PostHog 101!
+It all starts with getting the right people on your project. You can check the handbook for more information about our [small team structure](/handbook/company/small-teams) and [minimal approach to management](/handbook/company/management), or head to [our staff profiles](/team) to understand what our team looks like now.
+<br />
 
 <TutorialsSlider slugs={[
```

Comment:
To make it look good on the page, these should be 3 items or 6 I think.

Reply:
Yeah. Or my thinking was that maybe the last block is always a CTA, directing to the relevant handbook page.